src/core/rendering.js

```diff
@@ -770,23 +770,23 @@ p5.prototype.clearDepth = function(depth) {
  * function setup() {
  *   createCanvas(100, 100);
  *
- *   background(200);
+ *   background(180);
  *
  *   // Use the default blend mode.
  *   blendMode(BLEND);
  *
  *   // Style the lines.
  *   strokeWeight(30);
  *
- *   // Draw the blue line.
- *   stroke('blue');
+ *   // Draw the first line.
+ *   stroke('#1a85ff');
  *   line(25, 25, 75, 75);
  *
- *   // Draw the red line.
- *   stroke('red');
+ *   // Draw the second line.
+ *   stroke('#d41159');
  *   line(75, 25, 25, 75);
  *
- *   describe('A blue line and a red line form an X on a gray background.');
+ *   describe('A #1a85ff line and a #d41159 line form an X on a gray background.');
```

Comment:
In order to be more screen reader-accessible, I suggest we are more descriptive with the new color values instead of using hex codes like #d41159. For example, here we could say "A vibrant pink line and a bright blue line..."

This comment would be applicable to all the describe statements for the blend mode examples.

Reply:
Hey @aleannab, I completely understand the need to use color names, but I’m a bit uncertain about the best way to name these specific colors. Could you provide some guidance on how to approach this? I’d really appreciate your help!